_gtfobins/ssh.md

```diff
@@ -3,6 +3,20 @@ functions:
   execute-interactive:
     - description: Reconnecting may help bypassing restricted shells.
       code: ssh localhost $SHELL --noprofile --norc
+    - description: Spawn interactive shell through ProxyCommand option.
```

Comment:
Please replace `/bin/sh` instead of `/bin/bash` wherever possible.

Reply:
Also please use `$VAR` instead of `${VAR}` to be consistent with the rest.